tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -90,6 +91,9 @@ rootfs-image-tarball:
 rootfs-initrd-tarball:
 	${MAKE} $@-build
 
+rootfs-initrd-sev-tarball: kernel-sev-tarball
```

Comment:
We'd like to keep this sorted alphabetically, this move it to the before rootfs-initrd-tarball